Description: Add support for dash as default shell in Makefiles
 This patch allows the usage of dash as default SHELL when
 building scap-security-guide, avoiding advanced globbing.
Author: Philippe Thierry
Last-Update: 2017-02-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/RHEL/6/Makefile
+++ b/RHEL/6/Makefile
@@ -153,9 +153,10 @@
 	cp $(OUT)/$(ID)-$(PROD)-cpe-oval.xml $(DIST)/content
 	cp $(OUT)/$(ID)-$(PROD)-ds.xml $(DIST)/content
 	mkdir -p $(DIST)/tables
-	cp $(OUT)/table-*.{x,}html $(DIST)/tables
+	find $(OUT) -iname table-*.html -exec cp {} $(DIST)/tables \;
+	find $(OUT) -iname table-*.xhtml -exec cp {} $(DIST)/tables \;
 	mkdir -p $(DIST)/guide
-	cp $(OUT)/*-guide-*.html $(DIST)/guide
+	/bin/cp $(OUT)/*-guide-*.html $(DIST)/guide
 	cp $(OUT)/$(ID)-centos6-xccdf.xml $(DIST)/content
 	cp $(OUT)/$(ID)-centos6-ds.xml $(DIST)/content
 	cp $(OUT)/$(ID)-sl6-xccdf.xml $(DIST)/content
--- a/RHEL/7/Makefile
+++ b/RHEL/7/Makefile
@@ -146,7 +146,8 @@
 	cp $(OUT)/$(ID)-$(PROD)-cpe-oval.xml $(DIST)/content
 	cp $(OUT)/$(ID)-$(PROD)-ds.xml $(DIST)/content
 	mkdir -p $(DIST)/tables
-	cp $(OUT)/table-*.{x,}html $(DIST)/tables
+	find $(OUT) -iname table-*.html -exec cp {} $(DIST)/tables \;
+	find $(OUT) -iname table-*.xhtml -exec cp {} $(DIST)/tables \;
 	mkdir -p $(DIST)/guide
 	cp $(OUT)/*-guide-*.html $(DIST)/guide
 	cp $(OUT)/$(ID)-centos7-xccdf.xml $(DIST)/content
--- a/SUSE/11/Makefile
+++ b/SUSE/11/Makefile
@@ -143,7 +143,8 @@
 	cp $(OUT)/$(ID)-$(PROD)-cpe-oval.xml $(DIST)/content
 	cp $(OUT)/$(ID)-$(PROD)-ds.xml $(DIST)/content
 	mkdir -p $(DIST)/tables
-	cp $(OUT)/table-*.{x,}html $(DIST)/tables
+	find $(OUT) -iname table-*.html -exec cp {} $(DIST)/tables \;
+	find $(OUT) -iname table-*.xhtml -exec cp {} $(DIST)/tables \;
 	mkdir -p $(DIST)/guide
 	cp $(OUT)/*-guide-*.html $(DIST)/guide
 
--- a/SUSE/12/Makefile
+++ b/SUSE/12/Makefile
@@ -149,7 +149,8 @@
 	cp $(OUT)/$(ID)-$(PROD)-cpe-oval.xml $(DIST)/content
 	cp $(OUT)/$(ID)-$(PROD)-ds.xml $(DIST)/content
 	mkdir -p $(DIST)/tables
-	cp $(OUT)/table-*.{x,}html $(DIST)/tables
+	find $(OUT) -iname table-*.html -exec cp {} $(DIST)/tables \;
+	find $(OUT) -iname table-*.xhtml -exec cp {} $(DIST)/tables \;
 	mkdir -p $(DIST)/guide
 	cp $(OUT)/*-guide-*.html $(DIST)/guide
 	cp $(OUT)/$(ID)-centos7-xccdf.xml $(DIST)/content
